examples/AeroSharp.Examples/Maps/MapExampleData.cs

```diff
@@ -19,4 +20,35 @@ internal static class MapExampleData
         { "key9", "value9" },
         { "key10", "value10" }
     };
+
+/*
+ * A nested map example
+ * {
+ *   { "key1" : {
+*        { "key21", 999.99 },
+ *       { "key22", 999.99 }
+ *   },
+ *   { "key2" : {
+ *       { "a", 5 },
+ *       { "c", 3 }
+ *   }
+ * }
+ */
+    public static readonly Dictionary<string, object> nestedMap = new()
```

Comment:
Properties should be using [PascalCasing](https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members#names-of-properties:~:text=PascalCasing%20is%20always%20used%20for%20property%20names.)

```suggestion
    public static readonly Dictionary<string, object> NestedMap = new()
```

Reply:
My pinky doesn't like PascalCasing!  I should train it more with a better workout regimen.

Anywho, my miss, will get fixed